pkg/reconcilermanager/controllers/reconciler_base_test.go

```diff
@@ -640,8 +640,8 @@ func TestMountConfigMapValuesFiles(t *testing.T) {
 						MountPath: "/etc/config/foo/values.yaml",
 					}},
 					Env: []corev1.EnvVar{{
-						Name:  reconcilermanager.HelmValuesFileSources,
-						Value: filepath.Join("/etc/config/foo/values.yaml", reconcilermanager.HelmConfigMapRef),
+						Name:  reconcilermanager.HelmValuesFilePaths,
+						Value: filepath.Join("/etc/config/foo/values.yaml/helm_values_filepath"),
```

Comment:
this path is confusing to me. why is values.yaml a directory name?

Reply:
Fair point. 

I was trying to make the mount path unique by using a combination of the ConfigMap name and key. But that's not relevant anymore, because we've decided that we want to allow duplicate ConfigMap name/key tuples (since helm CLI allows duplicate valuesFiles). 

I will update this so that `values.yaml` ends up being the filename (which makes a lot more sense) and that the mountPath directory name is unique using the index instead.